### ISSUE TITLE
Find new point

### ISSUE THREAD

**Issue by thiagolims** [iocenter/FieldOpt-Research#18]
Implement pointNew() which is invoked in ensureImprovement() to find a new point.

**Comment by Bellout** [comment]
Should we just close this issue and move on, or should we create a test for this function, and then close the issue only after it has a test...? Or, can we set up an new issue called "Make test for pointNew()"?

**Comment by thiagolims** [comment]
Mathias, I guess you should close this issue only after creating a test for this function.

**Comment by Bellout** [comment]
Tested it manually against CG code, and performed ok.
